<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mainsignin</class>
 <widget class="QMainWindow" name="Mainsignin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Name :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_name">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>120</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Date of birth :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_year">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>220</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>year</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2001</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2002</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2003</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_month">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>220</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>month</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_day">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>220</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>day</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_gender">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>170</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>-----</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Male</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Female</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Gender :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>280</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>country :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_country">
    <property name="geometry">
     <rect>
      <x>331</x>
      <y>280</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>------</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Egypt</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Usa</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Canada</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Spain</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>France</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ðŸ”’ sign in </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>350</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>email:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_rules">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>450</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>I understand the terms  and conditions</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>400</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>password :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_email">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>350</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_password">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>400</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_confirm">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>500</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>confirm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cancel2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_back_to_mainwindow">
    <property name="geometry">
     <rect>
      <x>669</x>
      <y>60</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;-back</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
